src/index.js

```diff
@@ -1,11 +1,8 @@
 // @flow
-import resolve from './resolve';
+import resolve from './resolve'; // eslint-disable-line no-unused-vars
 
-export function createMessage(message) {
-    return `Sweesh, ${message}`;
-}
-
-console.log(createMessage('dude!'));
-
-// playing with this
-resolve([], 'test').catch(console.error);
+// not sure if we can use ES6 imports for yargs, so use require for now
+const argv = require('yargs') // eslint-disable-line no-unused-vars
+    .command(require('./commands/login'))
```

Comment:
Can we actually use [commandDir](https://github.com/yargs/yargs/blob/master/docs/advanced.md#commanddirdirectory-opts) here since we have everything in `/commands` ?